addons/mail/models/discuss/discuss_channel_member.py

```diff
@@ -336,7 +336,9 @@ def _rtc_join_call(self, store: Store = None, check_rtc_session_ids=None, camera
                 },
             )
         if len(self.channel_id.rtc_session_ids) == 1 and self.channel_id.channel_type != "channel":
-            self.channel_id.message_post(body=_("%s started a live conference", self.partner_id.name or self.guest_id.name), message_type='notification')
+            body = Markup("<div data-oe-type=\"call\" class=\"o_mail_notification\"></div>")
```

Comment:
I know we use `"` for python string, but this is a valid case for using `'` and not have the `\` 
```suggestion
            body = Markup('<div data-oe-type="call" class="o_mail_notification"></div>')
```